packages/components/Tabs/src/TabsItem.tsx

```diff
@@ -139,7 +142,7 @@ export const TabsItem = compose<TabsItemType>({
   slots: {
     root: View,
     stack: { slotType: View, filter: filterViewProps },
-    icon: { slotType: Icon as React.ComponentType },
+    icon: Platform.OS !== 'windows' ? { slotType: Icon as React.ComponentType } : React.Fragment,
```

Comment:
Why a Fragment?

Reply:
That check was added to make sure an icon is never rendered on UWP, but right now it seems unnecessary as it wont render due to a previous check on the component's state.

Will be removing the Fragment and changing the line back to:
```icon: { slotType: Icon as React.ComponentType },```